test/orders-test.js

```diff
@@ -1,24 +1,27 @@
-var _         = require('lodash');
-var assert    = require('assert');
-var ripple    = require('ripple-lib');
+/* eslint-disable new-cap */
```

Comment:
Should just be a find and replace to fix this right? If the same symbol is used in other files we could temporarily keep both names in the fixtures until all files are converted.